source/fuzz/call_graph.h

```diff
@@ -54,6 +66,19 @@ class CallGraph {
   // For each function id, stores the number of distinct functions that call
   // the function.
   std::map<uint32_t, uint32_t> function_in_degree_;
+
+  // Stores the ids of the functions in a topological order,
+  // in relation to the function calls, which are assumed to be recursion-free.
+  std::vector<uint32_t> functions_in_topological_order_;
+
+  // Computes a topological order of the functions in the graph, writing the
+  // result to |functions_in_topological_order_|. The function calls are assumed
+  // to be recursion-free.
+  void ComputeTopologicalOrderOfFunctions();
```

Comment:
See Google C++ style: https://google.github.io/styleguide/cppguide.html#Declaration_Order

We prefer to have methods before data members.

Reply:
Thanks, I moved method declarations to before data members